src/components/body/body.js

```diff
@@ -276,18 +268,22 @@ var Body = {
       if (!body) return;
 
       if (parentEl.isScene) {
-        el.object3D.quaternion.copy(body.quaternion);
-        el.object3D.position.copy(body.position);
+        q1.copy(body.quaternion);
+        e.setFromQuaternion(q1, 'YXZ');
+        el.setAttribute('rotation', { x: e.x * THREE.Math.RAD2DEG, 
+                                      y: e.y * THREE.Math.RAD2DEG, 
+                                      z: e.z * THREE.Math.RAD2DEG });
+        el.setAttribute('position', body.position);
```

Comment:
what version of A-Frame are you working with? I'd removed the `quaternion` component because A-Frame 0.8.0 will allow you to write to`object3D.*` safely. Using an euler will give gymbal lock issues (or i think that's the term for it). If we need this fixed pre-0.8.0 i may need to restore `quaternion`...

Reply:
I'm using a specific hash of aframe > 0.7.0 but should be able to be caught up to master soon. That said, it's pretty broken for <= 0.7.0, but its up to you if you want >= 3.0.0 to be compatible with <= 0.7.0.
Also, I'm not sure the way I'm using an euler will actually cause gimbal lock or not- I suspect not though because I'm not actually rotating the euler, just converting the quaternion to a euler.